kotlinpoet/src/test/java/com/squareup/kotlinpoet/UtilTest.kt

```diff
@@ -89,11 +89,48 @@ class UtilTest {
     assertThat("".isIdentifier).isFalse()
   }
 
+  @OptIn(ExperimentalStdlibApi::class)
   @Test fun escapeNonJavaIdentifiers() {
     assertThat("8startWithNumber".escapeIfNecessary()).isEqualTo("`8startWithNumber`")
     assertThat("with-hyphen".escapeIfNecessary()).isEqualTo("`with-hyphen`")
-    assertThat("with space".escapeIfNecessary()).isEqualTo("`with space`")
+    assertThat("with space".escapeIfNecessary()).isEqualTo("`with·space`")
     assertThat("with_unicode_punctuation\u2026".escapeIfNecessary()).isEqualTo("`with_unicode_punctuation\u2026`")
+
+    val generated = FileSpec.builder("a", "b")
+      .addFunction(FunSpec.builder("foo").apply {
+        addParameter("aaa bbb", typeNameOf<(Int) -> String>())
+        val arg = mutableListOf<String>()
+        addStatement(
+          StringBuilder().apply {
+            repeat(10) {
+              append("%N($it) + ")
+              arg += "aaa bbb"
+            }
+            append("%N(100)")
+            arg += "aaa bbb"
+          }.toString(),
+          *arg.toTypedArray()
+        )
+      }.build())
+      .build()
+      .toString()
+      .lines()
+      .filter { it.isNotBlank() }
```

Comment:
What's reason for stripping non-blank lines from the output?

Reply:
Sorry, just noticed your comment. This is strange, all other tests in the project don't have that issue. Is it just the trailing empty line that's causing problems? I think stripping non-empty lines is fine as a last resort, but ideally this shouldn't be needed.

Reply:
I read the error log again, and I found the error is not caused by test, but spotless.
Indent inserted by IntelliJ is reported as error by spotless :(
I run `gradlew spotlessKotlinApply` command, then it was fixed.
I deleted the code to remove empty lines.